src/servicemanager.rs

```diff
@@ -71,6 +73,7 @@ impl ServiceManager
 			send_bitcoind_request: Mutex::new(send_bitcoind_request),
 			send_events_gateway: Mutex::new(send_gateway_events),
 			our_service_pubkey: pubkey,
+			inclusion_proof: inclusion_proof,
```

Comment:
Good to integrate it like this, when the mainstay module is more encapsulated, it can be moved in `notaryd`.